etc/templates/error/502.html

```diff
@@ -0,0 +1,81 @@
+        </div>
+
+        <div id="login" class="standard_form inlined">
+	        <p>Site is currently down for maintenance. Please try again later.</p>
+
+        </div>
+
+        <div id="login-footer">
+	        <p>
+		        &copy; 2007-2014 University of Dundee &amp; Open Microscopy Environment<br>
```

Comment:
More generally, the question is should this page reflect the build date or the current date? Assuming OMERO.web is substituting the current date year, I would be inclined to use the same strategy.